packages/react/src/auto/AutoFormActionValidators.ts

```diff
@@ -44,19 +45,48 @@ export const validateAutoFormProps = (props: Parameters<typeof useAutoForm>[0])
 };
 
 export const validateTriggersFromMetadata = (metadata?: ActionMetadata | GlobalActionMetadata) => {
-  if (metadata) {
-    // When an action's API trigger is removed but the referenced API client still uses the old api-triggered action, the presence of the trigger shall be checked in the metadata
-
-    const triggersAsArray =
-      (metadata.__typename === "GadgetGlobalAction"
-        ? metadata.triggers
-        : metadata.__typename === "GadgetModel"
-        ? metadata.action.triggers
-        : []) ?? [];
-
-    const hasApiTrigger = triggersAsArray.some((trigger) => trigger.specID === GadgetApiTriggerSpecId);
-    if (!hasApiTrigger) {
-      throw new Error(MissingApiTriggerErrorMessage);
-    }
+  if (!metadata) {
+    return;
+  }
+  // When an action's API trigger is removed but the referenced API client still uses the old api-triggered action, the presence of the trigger shall be checked in the metadata
+
+  const triggersAsArray =
+    (metadata.__typename === "GadgetGlobalAction"
+      ? metadata.triggers
+      : metadata.__typename === "GadgetModel"
+      ? metadata.action.triggers
+      : []) ?? [];
+
+  const hasApiTrigger = triggersAsArray.some((trigger) => trigger.specID === GadgetApiTriggerSpecId);
```

Comment:
btw I think this might be the source of another issue I've seen with using autoform for auth: the special auth triggers are api triggers but don't have this specific spec id

Reply:
May I follow up on allowing API auth triggers separately to prevent this `findBy` updated from getting too large?

Reply:
yah of soruce; sorry this was just an fyi